### ISSUE TITLE
before_filter deprecated

### ISSUE THREAD

**Issue by atz** [projecthydra/curation_concerns#908]
```
DEPRECATION WARNING: 
before_filter is deprecated and will be removed in Rails 5.1. 
Use before_action instead. 
(called from include at .../app/controllers/curation_concerns/single_use_links_viewer_controller.rb:3)
```

**Comment by jcoyne** [comment]
This is already fixed in hydra-batch-edit pending merge of two PRs

**Comment by jcoyne** [comment]
Also one PR in hydra-head: https://github.com/projecthydra/hydra-head/pull/369

**Comment by jcoyne** [comment]
Closed by hydra-batch-edit 2.1.0 and hydra-head 10.2.0
